va/caa.go

```diff
@@ -218,12 +215,6 @@ func (va *ValidationAuthorityImpl) processRemoteCAAResults(
 	} else if bad > va.maxRemoteFailures {
 		modifiedProblem := *firstProb
 		modifiedProblem.Detail = "During secondary CAA checking: " + firstProb.Detail
-		// If the primary result was OK and there were more failures than the allowed
-		// threshold increment a stat that indicates this overall validation will have
-		// failed.
-		if primaryResult == nil {
```

Comment:
I wound up keeping the metrics. The earlier version of this PR was incorrect in removing them. Since the metrics are incremented when `primaryResult == nil`, and `primaryResult` is _always_ nil, the correct refactoring was to unconditionally increment them rather than delete them.

Reply:
Good catch.